docs/guides/tutorials/nextjs_app_router.rst

```diff
@@ -0,0 +1,627 @@
+
+Updating the homepage
+---------------------
+
+Let's start by implementing a simple homepage for our blog application using
+static data. Replace the contents of ``app/page.tsx`` with the following.
+
+.. code-block:: tsx
+
+  // app/page.tsx
+
+  import Link from 'next/link'
+
+  type Post = {
+    id: string
+    title: string
+    content: string
+  }
+
+  export default async function Home() {
+    const posts: Post[] = [
+      {
+        id: 'post1',
+        title: 'This one weird trick makes using databases fun',
+        content: 'Use EdgeDB',
+      },
+      {
+        id: 'post2',
+        title: 'How to build a blog with EdgeDB and Next.js',
+        content: "Let's start by scaffolding our app with `create-next-app`.",
+      },
+    ]
+
+    return (
+      <div className="container mx-auto p-4 bg-black text-white">
+        <h1 className="text-3xl font-bold mb-4">Posts</h1>
+        <ul>
+          {posts.map((post) => (
+            <li
+              key={post.id}
+              className="mb-4"
+            >
+              <Link
+                href={`/post/${post.id}`}
+                className="text-blue-500"
+              >
+                {post.title}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+    )
+  }
+
+
+After saving, Next.js should hot-reload, and you should see the updated
+homepage in your browser.
+
+Initializing EdgeDB
+-------------------
+
+Now let's spin up a database for the app. First, install the ``edgedb`` CLI.
+
+**Linux or macOS**
+
+.. code-block:: bash
+
+    $ curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
+
+**Windows Powershell**
+
+.. code-block:: powershell
+
+    PS> iwr https://ps1.edgedb.com -useb | iex
+
+Then check that the CLI is available with the ``edgedb --version`` command. If
+you get a ``Command not found`` error, you may need to open a new terminal
+window before the ``edgedb`` command is available.
+
+Once the CLI is installed, initialize a project from the application's root
+directory. You'll be presented with a series of prompts.
+
+.. code-block:: bash
+
+  $ edgedb project init
+  No `edgedb.toml` found in `~/nextjs-blog` or above
+  Do you want to initialize a new project? [Y/n]
+  > Y
+  Specify the name of EdgeDB instance to use with this project [default:
+  nextjs-blog]:
+  > nextjs-blog
+  Checking EdgeDB versions...
+  Specify the version of EdgeDB to use with this project [default: 2.x]:
+  >
+  ┌─────────────────────┬──────────────────────────────────────────────┐
+  │ Project directory   │ ~/nextjs-blog                                │
+  │ Project config      │ ~/nextjs-blog/edgedb.toml                    │
+  │ Schema dir (empty)  │ ~/nextjs-blog/dbschema                       │
+  │ Installation method │ portable package                             │
+  │ Start configuration │ manual                                       │
+  │ Version             │ 2.x                                          │
```

Comment:
Maybe show the current version (`4.5+641a8f3`) or `x.x` to make it feel more evergreen?
```suggestion
  │ Version             │ x.x                                          │
```

Reply:
Oh, same feedback up on line 154